src/styles/index.scss

```diff
@@ -1,3 +1,275 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
 }
+
+.catalog {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+}
+
+/* #region-Header */
+$linkColor: #060b35;
+$activ: #00acdc;
+
+.nav {
+  display: flex;
+  background: #fff;
+  height: 60px;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+  gap: 20px;
+}
+
+.nav :last-child {
+  margin-right: 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: $linkColor;
+}
+
+.is-active {
+  color: $activ;
+  position: relative;
+  padding: 22.5px;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: $activ;
+  height: 4px;
+  border-radius: 8px;
+  right: 0;
+  left: 0;
+  position: absolute;
+  bottom: 0;
+}
+
+.img {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav__link--laptops:hover {
+  color: $activ;
+  padding: 22.5px;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+/* #endregion */
+
+/* #region-Main */
+
+.main {
+  display: grid;
+
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: 1fr 1fr;
+
+  margin: 50px 40px;
+  grid-gap: 48px 46px;
+
+  justify-content: center;
+
+  /* #region-media-main */
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 778px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  /* #endregion */
+}
+
+.card {
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: solid #f3f3f3 1px;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The hover effect on `.card` elements scales them to 1.2 times their size. This could cause layout shifts or overlap with other elements, especially in a grid layout. Consider using a smaller scale factor or adding a transition to adjust the layout during hover.